api/build.gradle

```diff
@@ -20,25 +20,21 @@ buildscript {
     // External properties on the default project. Values declared in ext blocks
     // outside of the buildscript block aren't usable here.
     ext {
+        // Note: Spring versions are handled by plugins.
+        // See the plugins and dependencyManagement blocks below
+
         GAE_VERSION = '2.0.31'
         GOOGLE_AUTH_LIBRARY_VERSION = '1.30.1'
         GSON_VERSION = '2.11.0'
         HIBERNATE_VERSION = '6.5.2.Final'
         JACKSON_VERSION = '2.18.2'
         LIQUIBASE_VERSION = '4.29.2'
-        // 1.5.7+ results in a runtime error:
-        // Logging system failed to initialize using configuration from 'null'
-        //  java.lang.NoSuchMethodError: 'java.lang.Object ch.qos.logback.classic.LoggerContext.getConfigurationLock()'
-        LOGBACK_VERSION = '1.5.6'
```

Comment:
The Spring plugin pulls in a compatible logback automatically.  This PR brings it to 1.5.12.